modules/azure-control-plane-sizing.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/azure-control-plane-sizing-for-control-plane-to-be-performant.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="azure-control-plane-sizing_{context}"]
+= Recommended Control Plane Sizing for Azure
+
+For Azure, It is recommended to use Control Plane nodes with "Standard_D16s_v3" instances for improved performance and stability, as using "Standard_D8s_v3" instances led to firing critical alerts such as etcdGRPCRequestsSlow. 
```

Comment:
We might want to mention the the cluster size and load used in the prow cluster for reference. Thoughts?